bedrock/firefox/views.py

```diff
@@ -281,6 +282,14 @@ def fx_home_redirect(request):
     return HttpResponseRedirect(reverse('firefox.new'))
 
 
+def choose(request):
+    if waffle.switch_is_active('tracking-protection-redirect'):
```

Comment:
We normally make the state we want in prod now be the "off" state so that we don't have to make sure the flag is created and "ON" in all of the prod instances before we push.